docs/getting_started.md

```diff
@@ -112,18 +112,16 @@ Importing a key:
 
 ### Signing a message using RSA
 
-Mbed Crypto provides support for encrypting, decrypting, signing and verifying messages using public key signature algorithms (such as RSA or ECDSA).
+Mbed Crypto supports encrypting, decrypting, signing and verifying messages using public key signature algorithms, such as RSA or ECDSA.
 
-Prerequisites for performing asymmetric signature operations:
-* Initialize the library with a successful call to `psa_crypto_init`.
+**Prerequisites to performing asymmetric signature operations:**
```

Comment:
I see a mix of `Prerequisites to` and `Prerequisites for` in this guide. Should we pick one usage and use it consistently?